### ISSUE TITLE
Is there a way to update the fxlauncher on the fly?

### ISSUE THREAD

**Issue by gabrielpaim** [edvin/fxlauncher#118]
Hi guys,

This may sound unnecessary as the launcher process is quite generic, but considering some customization such as the CustomUI and [this issue](https://github.com/edvin/fxlauncher/issues/116), this could be required. How could I update the fxlauncher itself? Naturally, during the app lifecycle this is not allowed, but do you have any suggestion for that? 

Thanks!

**Comment by gabrielpaim** [comment]
Ok, thank you Edvin for the super fast respose :)
By the way, are you organizing your plans for this mentioned reboot/2.0 (https://github.com/edvin/fxlauncher/issues/37) somewhere?

**Comment by edvin** [comment]
No, I'm not as I'm waiting to see what becomes of the Graviton Browser. Knowing Mike Hearn this might be  a far better solution with a lot more bells and whistles :)

https://graviton-browser.readthedocs.io/en/latest/

**Comment by gabrielpaim** [comment]
Ok, thanks again Edvin. I'm closing the issue. :)

**Comment by gabrielpaim** [comment]
Hi Edvin,

let me quickly reopen this issue to ask you if you happened to check [this](https://github.com/graviton-browser/graviton-browser/releases/tag/r1), as you mentioned it in your last message. It seems really interesting, indeed.

So far, we're shipping an App (that is still in beta) to our users with FxLauncher (thank you!), but we needed to implement a way to update the launcher itself through an outside native script (shell, batch). Although it works well, we needed to set a hard requirement to not install the application in a folder without write permission. It seems that this decision may become an issue for us in a near future. Still, I really would like to known what are your thoughts on this, if you are still considering this reboot/2.0.

**Comment by edvin** [comment]
Graviton looks very promising, and is written by an extremely smart guy :) I want to investigate the need for FXLauncher 2.0 in light of Graviton, but I haven't had time to do so yet I'm afraid.
